lightning/mydump/region.go

```diff
@@ -57,10 +57,13 @@ func (rs regionSlice) Swap(i, j int) {
 	rs[i], rs[j] = rs[j], rs[i]
 }
 func (rs regionSlice) Less(i, j int) bool {
-	if rs[i].FileMeta.Path == rs[j].FileMeta.Path {
-		return rs[i].Chunk.Offset < rs[j].Chunk.Offset
+	if rs[i].FileMeta.SortKey != rs[j].FileMeta.SortKey {
+		return rs[i].FileMeta.SortKey < rs[j].FileMeta.SortKey
 	}
-	return rs[i].FileMeta.Path < rs[j].FileMeta.Path
+	if rs[i].FileMeta.Path != rs[j].FileMeta.Path {
+		return rs[i].FileMeta.Path < rs[j].FileMeta.Path
+	}
```

Comment:
the ordering doesn't look correct, as `db.z.0.sql` will be sorted before `db.a.100.sql`. 

but this function seems totally unused, so let's just delete the whole `regionSlice` type 🙂.

Reply:
This would happen only if `db.z.0.sql` and `db.a.100.sql` are routing to the same table. And If files with different perfix are routing to the same table, sort by sort-key should be the correct way to order them, because sort-key is designed for this

Reply:
BTW, this sort is useless because at this moment, each chunk's handle alloction is already determined. Which chunk restore first will almost make no difference.